### DISCUSSION TITLE
How to use sidebar with children elements?

### DISCUSSION THREAD

**Discussion by Arturexe** [vuejs/vitepress#2123] [Q&A]
I'm trying to represent my folder tree in the sidebar but when I click on one of the `Group` parent elements, the children elements are not rendered. 

 [![enter image description here][1]][1]

Another thing that's weird is that the `Group` elements have no `cursor: pointer` css property, like you would expect on a clickable object. This is my `.vitepress/config.ts`

    export default {
      title: "blub docs",
      description: "blub",
      themeConfig: {
        nav: [
          { text: 'Home', link: '/' },
          { text: 'Examples', link: '/markdown-examples' }
        ],
        sidebar: {
          '/': [
            {
              text: 'Group 1',
              collapsible: true,
              children: [
                '/group1/page1.md',
                '/group1/page2.md',
              ],
            },
            {
              text: 'Group 2',
              collapsible: true,
              children: [
                '/group2/page1.md',
                '/group2/page2.md',
              ],
            },
          ],
        }
      }
    }

I've also commented out the `layout` from `root/index.md` so that the sidebar is visible on the landing page (if that is relevant):

    [//]: # (layout: home)

This is the default layout of vitepress. I've implemented a custom layout but ended up having exactly the same issue.

  [1]: https://i.stack.imgur.com/kdc98.png

**Comment by brc-dd** [comment]
That looks like vuepress/vitepress v0 syntax. Refer the docs for latest syntax: https://vitepress.dev/reference/default-theme-sidebar
